### DISCUSSION TITLE
pyMOR Community Meeting 2023-03-22

### DISCUSSION THREAD

**Discussion by lbalicki** [pymor/pymor#1984] [Announcements]
The next pyMOR Community Meeting will take place Wednesday, March 22, at 14:00 CET and 9:00 EST, online at:

- link: https://bbb.mpi-magdeburg.mpg.de/meeting/pymor
- password: THaEo

Anyone interested in pyMOR can join.

Currently on the agenda are:

- pyMOR School 2023 planning (@pmli)
- pyMOR 2023.1 and 2022.2.1 release schedule (@pmli)
- description and features in readme and docs, getting started (@pmli)
- public docker images (@sdrave)

**Comment by pmli** [comment]
Minutes:

- pyMOR School 2023 planning
  - Berlin
  - the start of the semester: Oct 16 in TU Berlin, Oct 9 in Muenster
  - time of the school: before the semester at TU Berlin would be good
  - we should reserve a room for a week
  - registration tool (MPI Indico?)
  - registration fee, funding, taxes?
  - maybe no hybrid option
- pyMOR 2023.1 and 2022.2.1 release schedule
  - we should settle on the release process
  - tentative release day: June 29
  - 2022.2.1: should add ipywidgets fixes (https://github.com/pymor/pymor/pull/1949)
- description and features in readme and docs, getting started
  - PRs for more features are welcome (e.g., randomized methods)
  - more words about MOR in the description might be good
  - should update the docs and homepage
  - getting started: a few minimal but complete examples, link to tutorials
  - we should work on the documentation (https://github.com/pymor/pymor/discussions/1990)
- public docker images
  - Dockerhub is getting rid of free teams
  - we have a pyMOR team on Dockerhub
  - there is an open source program on Dockerhub, but we could convert it to a user account, which should be sufficient for us
  - otherwise, we could remove public images or publish them on GitHub
  - we'll try to get a user account

Next meeting: 2023-04-12, 12:00 UTC (14:00 CEST, 8:00 EDT)
Next moderator: @artpelling

**Reply by drittelhacker** [reply]
MPI Indico for2023 school is perfectly fine. As soon as the dates are fixed, I will create the event and equip @artpelling with the required permissions.
